<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevInfoWidget</class>
 <widget class="QWidget" name="DevInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="_unpairPushButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>所有设备取消配对</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_connInfoLabel">
       <property name="text">
        <string>本节列出您的电脑连通性选项信息，连同任何Bodystat信息设备相连。</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="_sacnDevPushButton">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>扫描新设备</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_reloadDevPushButton">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>重载设备信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="_bluetoothInfoLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>蓝牙无线电信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_bluetoothInfLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>蓝牙无线电信息</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="_bluetoothLabel">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>72</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="_radioManufLabel">
             <property name="text">
              <string>无线电制造商：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_radioHwVerLabel">
             <property name="text">
              <string>无线电硬件版本：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_radioSwVerLabel">
             <property name="text">
              <string>无线电软件版本：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="_bodystatDevInfoLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Bodystat设备信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_bodystatDevInfLabel">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这部分查询Bodystat设备（通过蓝牙，无线或网络）来验证连接正常工作。Bodystat设备详细信息如下所示。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="_devLabel">
             <property name="minimumSize">
              <size>
               <width>129</width>
               <height>102</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_progressLabel">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="_firmVerLabel">
             <property name="text">
              <string>固件版本：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_serialNoLabel">
             <property name="text">
              <string>序列号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_calibDateLabel">
             <property name="text">
              <string>校准日期：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="_devModelLabel">
             <property name="text">
              <string>设备模型：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="_msgLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
